python/python/glide/async_commands/core.py

```diff
@@ -1042,6 +1042,24 @@ async def llen(self, key: str) -> int:
         """
         return cast(int, await self._execute_command(RequestType.LLen, [key]))
 
+    async def strlen(self, key: str) -> int:
+        """
+        Get the length of the string value stored at key.
+        See https://redis.io/commands/strlen/ for details.
+
+        Args:
+            key (str): The key
+
+        Returns:
+            int: The length of the string value stored at key.
+                If `key` does not exist, it is interpreted as an empty list and 0 is returned.
```

Comment:
```suggestion
                If `key` does not exist, it is treated as an empty string and 0 is returned.
```

Reply:
there is no difference between 'interpreted' and 'treated' between

Reply:
true, just trying to be consistent between docs from different wrappers